doc/changelog.rst

```diff
@@ -170,12 +170,15 @@ Breaking Changes in 4.0
   are passed to the server as-is rather than the previous behavior which
   substituted in a projection of ``{"_id": 1}``. This means that an empty
   projection will now return the entire document, not just the ``"_id"`` field.
-- :class:`~pymongo.mongo_client.MongoClient` now raises a :exc:`~pymongo.errors.ConfigurationError`
-  when more than one URI is passed into the ``hosts`` argument.
+- :class:`~pymongo.mongo_client.MongoClient` now raises a
+  :exc:`~pymongo.errors.ConfigurationError` when more than one URI is passed
+  into the ``hosts`` argument.
 - :class:`~pymongo.mongo_client.MongoClient`` now raises an
   :exc:`~pymongo.errors.InvalidURI` exception
   when it encounters unescaped percent signs in username and password when
   parsing MongoDB URIs.
+- Removed the `disable_md5` parameter for :class:`~gridfs.GridFSBucket` and
+  :class:`~gridfs.GridFS`. MD5 checksums are now always disabled in GridFS.
```

Comment:
Instead of saying "checksums are now always disabled" I think you should say something like "gridfs no longer generates a checksum" and link to the guidance in the upgrade docs.

Reply:
Done.